### DISCUSSION TITLE
amazon sp-api inbound upload flow

### DISCUSSION THREAD

**Discussion by guoshuaichao** [saleweaver/python-amazon-sp-api#1097] [FAQ]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Desktop (please complete the following information):**

 - OS: [e.g. Linux/Mac/Windows]

**Additional context**

**Comment by guoshuaichao** [comment]
How to upload carton size and carton information

**Reply by chocobn69** [reply]
I'm sorry but it does not look like a python lib bug to me
